v3/profiles/todo.go

```diff
@@ -0,0 +1,5 @@
+package profiles
```

Comment:
Yeah, sorry, if you have a package with nothing in it but test files then `golangci-lint` vomits uncontrollably. We'll have to delete this once our first profile lands.